ui/src/components/UHGroupingsInfo.tsx

```diff
@@ -0,0 +1,67 @@
+import Image from 'next/image';
+
+const UHGroupingsInfo = ({
+    size
+}: {
+    size?: 'lg' | 'default';
+}) => {
+    const color = size === 'lg' ? 'text-text-color' : 'text-uh-black';
+    const textSize = size === 'lg' ? 'text-[1.2rem]' : 'text-base';
+
+    const items = [
```

Comment:
Let's call this variable `infoItems` for clarity. For the `.map` function, make sure to rename to `item` to `infoItem`, thanks!